scripts/install.sh

```diff
@@ -25,3 +25,11 @@ if [[ "${TOX_ENV}" == "pypy" ]]; then
     pyenv install pypy-2.6.0
     pyenv global pypy-2.6.0
 fi
+
+if [[ "${TOX_ENV}" == "gae" && ! -d $HOME/google_appengine ]]; then
```

Comment:
Why aren't we using `${GAE_PYTHONPATH}`?

Aside: I am doing a bash code review. Weird!